### PR TITLE
Pass the theme to the tsiCharts

### PR DESCRIPTION
**Pull request by spryor**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Make the tsiCharts theme change with the app theme

### CODE REVIEW

File: src/components/pages/dashboard/dashboard.js [hunk 1]
```diff
@@ -385,6 +387,7 @@ export class Dashboard extends Component {
               telemetry={telemetry}
               isPending={telemetryIsPending}
               error={deviceGroupError || telemetryError}
+              theme={theme}
               colors={chartColorObjects}
```
Comment (mechaffin) [inline src/components/pages/dashboard/dashboard.js]: Shouldn't the colors come from the theme?

File: src/components/pages/maintenance/maintenance.js [hunk 2]
```diff
@@ -150,7 +150,7 @@ export class Maintenance extends Component {
   onTimeIntervalChange = (timeInterval) => this.setState({ timeInterval }, () => this.getData());
 
   render() {
-    const { rulesEntities, deviceEntities, rulesIsPending, t, history } = this.props;
+    const { rulesEntities, deviceEntities, rulesIsPending, theme, t, history } = this.props;
```
Comment (mechaffin) [inline src/components/pages/maintenance/maintenance.js]: At what point should these be on separate lines? I recall being asked to make separate lines when lists get long. I've been using 3 as a rule of thumb.

File: src/components/pages/dashboard/dashboard.js (hunk 1 above)
Comment (spryor) [reply src/components/pages/dashboard/dashboard.js]: They should but that is a larger item than this task. Currently, if you switch to the light theme, the tsi charts stay on the default dark theme where the legends and axis are white. So you can't see them, This PR just makes the charts readable when the light theme is turned on.

File: src/components/pages/maintenance/maintenance.js (hunk 2 above)
Comment (spryor) [reply src/components/pages/maintenance/maintenance.js]: This rule isn't strictly enforced but in the past we have been using character count as the cut off. I think the cutoff is 90 something chars. It is a judgment call though and I'm fine with this (currently 86 chars long anyway).

**Pull request merged**